src/reconcile/rec_col.c

```diff
@@ -56,19 +56,24 @@ __wt_bulk_insert_fix(WT_SESSION_IMPL *session, WT_CURSOR_BULK *cbulk, bool delet
 {
     WT_BTREE *btree;
     WT_CURSOR *cursor;
+    WT_PAGE_STAT ps;
     WT_RECONCILE *r;
     WT_TIME_WINDOW tw;
 
     r = cbulk->reconcile;
     btree = S2BT(session);
     cursor = &cbulk->cbt.iface;
+    WT_PAGE_STAT_INIT(&ps);
 
     WT_RET(__rec_col_fix_bulk_insert_split_check(cbulk));
     __bit_setv(
       r->first_free, cbulk->entry, btree->bitcnt, deleted ? 0 : ((uint8_t *)cursor->value.data)[0]);
     ++cbulk->entry;
     ++r->recno;
 
+    ps.row_count = 1;
```

Comment:
I don't understand why we need to set it to 1 first.

Reply:
I think we only need to aggregate when we try to write out the leaf page.

Reply:
Same for row store and column store in the bulk load case.

Reply:
I'm setting it to 1 to increment the row count when we call update. I was just trying to follow the places where we update the time aggregate as well, but we can probably discuss this more.